great_expectations/profiler/domain_builder/domain_builder.py

```diff
@@ -18,7 +18,7 @@ def get_domains(
         *,
         validator: Optional[Validator] = None,
         batch_ids: Optional[List[str]] = None,
-        include_batch_id: Optional[bool] = False,
+        include_batch_id: Optional[bool] = True,
```

Comment:
The more I think about the multibatch case, the more I think including a `batch_id` is mandatory, even for single batch cases just to be consistent.

Reply:
@anthonyburdi Let us then add to our discussion "whether or not we should remove the boolean flag" and always include it.  Thanks.

Reply:
Resolved -- removing the boolean and always including `batch_id` in `domain_kwargs`.